osu.Game/Extensions/WebRequestExtensions.cs

```diff
@@ -16,7 +18,7 @@ public static void AddCursor(this WebRequest webRequest, Cursor cursor)
         {
             cursor?.Properties.ForEach(x =>
             {
-                webRequest.AddParameter("cursor[" + x.Key + "]", x.Value.ToString());
+                webRequest.AddParameter("cursor[" + x.Key + "]", (x.Value as JValue)?.ToString(CultureInfo.InvariantCulture) ?? x.Value.ToString());
```

Comment:
I would also accept `FormattableString.Invariant($"{x.Value}")` but either works so I'm not going to split hairs.

Reply:
does that somehow get the value from the JToken in this case?

Reply:
It appeared to in my brief testing, yeah. Again not an issue though.